bot/plotting/config_generator.py

```diff
@@ -0,0 +1,205 @@
+import random
+import pybamm
+from experiment.experiment_generator import experiment_generator
+
+# possible chemistries for the bot
+chemistries = [
+    pybamm.parameter_sets.Ai2020,
+    pybamm.parameter_sets.Chen2020,
+    pybamm.parameter_sets.Marquis2019,
+    pybamm.parameter_sets.Yang2017,
```

Comment:
FYI Yang2017 is going to be removed soon (see [here](https://github.com/pybamm-team/PyBaMM/pull/1577))

Reply:
Thanks! Removing it